astroquery/_astropy_init.py

```diff
@@ -25,36 +25,9 @@
 
 if not _ASTROPY_SETUP_:  # noqa
```

Comment:
Seems a good opportunity to remove `_ASTROPY_SETUP_` as well.

Reply:
I am not sure. `astroquery` is still using `astropy-helpers`.

Reply:
Oh, right  :astonished:

Reply:
Yep, there were bigger fishes to fry than fixing what is not yet broken, but hopefully, we will have some capacity in the future to do some refactoring.